assets/app/views/_alerts.html

```diff
@@ -1,20 +1,32 @@
 <div ng-repeat="alert in alerts">
   <div ng-switch="alert.type">
-    <div ng-switch-when="error" class="alert alert-danger">
+    <div ng-switch-when="error" class="alert alert-danger" ng-class="{'alert-dismissable': alert.dismissable}">
+      <button ng-if="alert.dismissable" type="button" class="close" data-dismiss="alert" aria-hidden="true">
```

Comment:
Yeah, I'm asking more about Bootstrap `data-dismiss="alert"` because I don't know how it works. For instance, if I add another alert back with the same key, will it show up or stay dismissed from before?